.env.example

```diff
@@ -67,3 +67,5 @@ NOTIFY_API_KEY=f4k3_k3y
 FROM_EMAIL=test@example.com
 GOVUK_NOTIFY_PLAIN_EMAIL_TEMPLATE_ID=example-id
 EMAIL_FILE_PATH='/app/mail'
+USE_STREAMING=False
```

Comment:
Is this only used by the Django app? Maybe we don't need the new env variable in the non-Django environment list?

Reply:
It is. I didn't know you could set the env variables anywhere else! Where would a Django specific setting go?

Reply:
Hm what this is showing me is that our dev docs are not very clear on the purpose of the 4 different sample .env files. `.env.django` is specifically for settings for running the application with Django (https://github.com/i-dot-ai/redbox-copilot/blob/cab97583c7a4018fc8c07a6b9078e153980a49eb/.github/workflows/django-checks.yml#L36) but maybe my question is moot given that we're thinking of streamlining our env files.